tests/functional/spec/list_collector_variants_primary_person.spec.js

```diff
@@ -0,0 +1,121 @@
+import VariantBlockPage from "../generated_pages/list_collector_variants_primary_person/variant-block.page";
+import PrimaryPersonListCollectorPage from "../generated_pages/list_collector_variants_primary_person/primary-person-list-collector.page";
+import ListCollectorAddPage from "../generated_pages/list_collector_variants_primary_person/list-collector-add.page";
+import ListCollectorPage from "../generated_pages/list_collector_variants_primary_person/list-collector.page";
+import EditPersonPage from "../generated_pages/list_collector_variants_primary_person/list-collector-edit.page";
+import SubmitPage from "../generated_pages/list_collector_variants_primary_person/submit.page";
+import ThankYouPage from "../base_pages/thank-you.page.js";
+
+describe("List collector with variants primary person", () => {
+  describe("Given that person is lives in house", () => {
+    before("Load the survey", () => {
+      browser.openQuestionnaire("test_list_collector_variants_primary_person.json");
+      it("The user is asked questions about whether they live there", () => {
+        $(VariantBlockPage.yes()).click();
+        $(VariantBlockPage.submit()).click();
+        expect($(PrimaryPersonListCollectorPage.legend()).getText()).to.contain("Do you live here? (variant)");
+      });
+    });
+  });
+  describe("Given the user starts on the 'Do you like variant' question", () => {
```

Comment:
We usually go with respondent, rather than user

Reply:
This is the pattern i followed , it uses `user` 
https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/tests/functional/spec/list_collector_primary_person.spec.js

Reply:
Ah ok, fair enough